Update mobility-toolbox-js@1.6.3

oterral opened the pull request:
# How to

<!--  Please provide a test link and quick description how to see the change -->
This PR fix imports to prepare to use it with the last version of react-scripts  and change the RouteSchedule example

# Others

<!-- Thanks for the PR! Feel free to add or remove items if there are not necessary. -->

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

danji90 reviewed (changes requested):
@oterral There are still a couple of issues:
- The route schedule doesn't close when the a route is deselected via map click (if this is the desired behaviour we should at least add a close button on the schedule container)
- Sometimes when you switch from one route to another residual stations from the last schedule are rendered above the Perlenkette. I dunno if this bug was already there before though

oterral commented:
> @oterral There are still a couple of issues:
> 
> * The route schedule doesn't close when the a route is deselected via map click (if this is the desired behaviour we should at least add a close button on the schedule container)
Fixed
> * Sometimes when you switch from one route to another residual stations from the last schedule are rendered above the Perlenkette. I dunno if this bug was already there before though
Fixed but there i also a backend problem. I have contacted the devs.
